src/components/seo.js

```diff
@@ -25,32 +25,40 @@ function SEO({ description, lang, meta, keywords, title }) {
       meta={[
         {
           name: `description`,
-          content: metaDescription,
+          content: 'A place for Q&A and free mentorship',
```

Comment:
In an effort to keep the description the same across the site, `metaDescription` is defined on line 18 and derives the description from `gatsby-config.js`. I'd prefer we not hardcode that unless you think this is a better change. Thoughts?

Reply:
Resolved